src/js/jobs/sagas.js

```diff
@@ -47,5 +75,8 @@ export function* watchJobs() {
     yield takeEvery(CANCEL_JOB.REQUESTED, cancelJob);
     yield takeEvery(ARCHIVE_JOB.REQUESTED, archiveJob);
     yield takeLatest(WS_UPDATE_JOB, wsUpdateJob);
```

Comment:
This needs to update counts as well. What if a job changes from waiting to preparing?

Reply:
I assume your talking about `WS_UPDATE_JOB`? If so, it already does by pushing a get `getJobCount` action. Probably going to change how this works on next push though